src/server/licensing_func.c

```diff
@@ -694,8 +697,10 @@ init_licensing(struct work_task *ptask)
 		if (pbsndlist[i]->nd_attr[ND_ATR_LicenseInfo].at_val.at_long) {
 			licensing_control.licenses_total_needed += pbsndlist[i]->nd_attr[ND_ATR_LicenseInfo].at_val.at_long;
 		} else {
-			set_node_lic_info_attr(pbsndlist[i]);
-			licensing_control.licenses_total_needed += pbsndlist[i]->nd_attr[ND_ATR_LicenseInfo].at_val.at_long;
+			if (pbsndlist[i]->nd_lic_info != NULL) {
+				set_node_lic_info_attr(pbsndlist[i]);
+				licensing_control.licenses_total_needed += pbsndlist[i]->nd_attr[ND_ATR_LicenseInfo].at_val.at_long;
```

Comment:
Please use get_attr_l instead of accessing it directly.

Reply:
done